src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,19 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence.
+    //[category: Creating generators]
+    [<CompiledName("Shuffle")>]
+    let shuffle xs =
+        gen {
+            let g = choose (-922337203, 922337203)
```

Comment:
True, these magic numbers result to a high(er) cognitive load. They could as well be bound to names, hypothetically `minBound` and `maxBound`.